logging/logger.js

```diff
@@ -1,40 +1,12 @@
 ﻿var winston = require('winston');
```

Comment:
I think we should put some thought into what makes sense for logging. you mentioned that docker auto creates logs based on stdout and stderr, but will we be always be running docker when we run the app locally? Is there any value in  still logging to a file (easier log access during local development, daily rotated files in production, pretty colors in the console, etc.)? If so, we might want to keep more than just the silly transport. If not, then we could probably just delete logger and remove winston completely